doc/source/User_guide/pyaedt_extensions_doc/project.rst

```diff
@@ -0,0 +1,49 @@
+Import Nastran
+==============
+
+Import Nastran or STL file in any 3D modeler application. The user can also preview the imported file and decimate it before import it.
```

Comment:
```suggestion
Import Nastran or STL file in any 3D modeler application. The user can also preview the imported file and decimate it prior to import.
```

Reply:
You can import a Nastran or STL file in any 3D modeler application. You can also preview ....

The Google dev doc style guide says to use "app" rather than application, but I'm OK with whatever as long as use the term consistently throughout the doc.